web-ui/session-helpers.mjs

```
@@ -212,6 +254,11 @@ export async function loadSessions(api, options = {}) {
         if (loadSucceeded) {
             this.sessionsLoadedOnce = true;
         }
+        const pendingOptions = this.__sessionPendingLoadOptions || null;
+        this.__sessionPendingLoadOptions = null;
+        if (pendingOptions) {
+            return loadSessions.call(this, api, pendingOptions);
```

Biome (2.4.10)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)